### PR TITLE
Bump phpseclib/phpseclib from 2.0.25 to 2.0.30

### PR DESCRIPTION
**Pull request by ChristophWurst**
https://github.com/nextcloud/3rdparty/pull/592

### CODE REVIEW

**Review by ChristophWurst — COMMENTED** [review]
:+1:

**Comment by rullzer** [comment]
Ok got it. So the new phpseclib fails on parsing the root.crt since it contains more than 1 cert.
Before it just only took the first one.

We still ahve to fix:

* the check app
* the check core

File: lib/private/Installer.php [hunk 1]
```diff
@@ -215,6 +215,18 @@ public function updateAppstoreApp($appId, $allowUnstable = false) {
...
 	}
 
+	/**
+	 * Split the certificate file in individual certs
+	 *
+	 * @param string $cert
+	 * @return string[]
+	 */
+	private function splitCerts(string $cert): array {
+		preg_match_all('([\-]{3,}[\S\ ]+?[\-]{3,}[\S\s]+?[\-]{3,}[\S\ ]+?[\-]{3,})', $cert, $matches);
```
Comment (nickvergessen) [inline lib/private/Installer.php:225]: Luckily we have tests that confirm this  🙈

**Comment by ChristophWurst** [comment]
I think drone got stuck

**Comment by rullzer** [comment]
yeah restarted it so lets see

**Comment by rullzer** [comment]
So now different tests
So lets do this.

**Review by rullzer — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]
Looks good 👍

**Pull request merged**